src/legacy/core_plugins/embeddable_api/public/context_menu_actions/build_eui_context_menu_panels.ts

```diff
@@ -104,12 +104,10 @@ function convertPanelActionToContextMenuItem({
     'data-test-subj': `embeddablePanelAction-${action.id}`,
   };
 
-  if (action.getHref(actionContext) === undefined) {
-    menuPanelItem.onClick = () => {
-      action.execute(actionContext);
-      closeMenu();
-    };
-  }
+  menuPanelItem.onClick = () => {
```

Comment:
Change was necessary or the link actions that navigated the user to another page would result in the pop up menu still showing.